movies/movies

```diff
@@ -158,28 +166,16 @@ getConfiguredPython || exit 1
 getConfiguredClient || exit 1
 checkInternet || exit 1 # check if we have a valid internet connection if this isnt true the rest of the script will not work so stop here
 
-while getopts "uvh" opt; do
-  case $opt in
-    \?)
-      echo "Invalid option: -$OPTARG" >&2
-      exit 1
-    ;;
-    h)
-      usage
-      exit 0
-    ;;
-    v)
-      echo "Version $currentVersion"
-      exit 0
-    ;;
-    u)
-      update
-      exit 0
-    ;;
-    :)
-      echo "Option -$OPTARG requires an argument." >&2
-      exit 1
-    ;;
+while getopts 'ud:hv' flag; do
```

Comment:
I prefer it to actually show that error that is exactly what I use the colon for as well. But the way it is set up right now I am not sure that would happen. In all my other scripts the way that was added was putting another option in the getops 
```bash
 :)
     echo "Option -$OPTARG requires an argument." >&2
     exit 1
   ;;
```